cmd/network/setup_linux.go

```diff
@@ -43,33 +45,22 @@ var (
 )
 
 const (
-	nsenter            = "/usr/bin/nsenter"
-	unshare            = "/usr/bin/unshare"
-	vsockHandshakePort = 6669
-	vsockDialPort      = 6656
-	defaultTapDevice   = "eth0"
+	nsenter              = "/usr/bin/nsenter"
+	unshare              = "/usr/bin/unshare"
+	vsockHandshakePort   = 6669
+	vsockDialPort        = 6656
+	defaultTapDevice     = "eth0"
+	defaultNSVeth        = "veth0"
+	rancherDesktopNSVeth = "veth1"
```

Comment:
Do we need to find names dynamically for these? (Other distros might have created some veths already)

Reply:
Dynamically, it would be ideal but that might lead to many lookup and timing issues. I renamed them to 
```
	defaultNSVeth        = "veth-rd0"
	rancherDesktopNSVeth = "veth-rd1"
```
If we come across any issues we will tackle it then.